models/sklearn/classification/report_files_export.py

```diff
@@ -0,0 +1,26 @@
+import os
+from datetime import datetime
+from termcolor import colored
+from utils import load_yaml, FindCreateDirectory, TrainingProcesses
+
+
+def export_report(config, name, report, filename, train_class, exports_path):
+    exports_dir = "{}_{}".format(config.get("exports_directory"), train_class)
+    reports_path = FindCreateDirectory(exports_path, os.path.join(exports_dir, "reports")).inspect_directory()
+    # take current date and convert to string
+    now = datetime.now()
+    datetime_str = now.strftime("%Y-%m-%d")
+    datetime_str_verbose = now.strftime("%Y-%m-%d, %H:%M:%S")
+    print("Creating report file..")
+    with open(os.path.join(reports_path, "{}.txt".format(filename)), 'w+') as file:
+        file.write("{}".format(name))
+        file.write('\n')
```

Comment:
you can put multiple `\n`s in a single write command if you want. Don't mix " and '
In the line above, if you just have name, you can probably just use fp.write(name) unless there's a specific reason for doing the format

Reply:
I just used the `"{}".format(name)` for safety, because someone in the future could add a number for `name`. However, I changed it to `str(name)`.